frontend/components/Status.tsx

```diff
@@ -0,0 +1,65 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { ITask, Ilist } from "../src/App";
+
+function Status(props: {
+  submited: boolean;
+  setSubmited: (Submit: boolean) => void;
+  list: ITask[];
```

Comment:
I also find the name `list` unclear. Many things in the codebase are 'lists'. A more explicit name would be `taskList` or simply `tasks`.